sysutils/dssystem/files/dssystem.in

```diff
@@ -31,6 +31,16 @@ dssystem_prestart()
     IP=`cat /root/dhcpip`
     sed -i.bak -E "s|codebase=\"(.+)\"|codebase=\"http://$IP/asigra/\"|" /usr/local/www/asigra/DSOP.jnlp
     rm -f /usr/local/www/asigra/DSOP.jnlp.bak
+
+    dssys_file="/etc/dssys.cfg"
+    # We make sure that the port is not added to the IP Address
+    ip=$(echo ${IP} | cut -d':' -f1)
+    grep -q "External IP Addr :" "${dssys_file}"
```

Comment:
Well, I was thinking we could be more bullet proof than that, thats why I asked about the number of spaces. Using 1 vs 20 essentially has the exact same problem.

Can we please make it behave so it doesnt matter? e.g.
^External IP Addr.*: